### PR TITLE
Document why OASIS needs OASISRepoURLs

### PR DESCRIPTION
**Pull request by brianhlin**
@DrDaveD does this look right?

### CODE REVIEW

**Comment by DrDaveD** [comment]
That does look right.  If you're trying to be complete, however, oasis-login also reads the XML data for OASIS/Managers and also depends on OASIS/UseOASIS via keywords in the URL: /vosummary/xml?summary_attrs_showoasis=on&oasis=on&oasis_value=1.

**Comment by matyasselmeci** [comment]
The rest of the text doesn't have information that detailed, so I think this is good enough. Feel free to link to Dave's comment though :) https://github.com/opensciencegrid/topology/pull/248#issuecomment-432360176

**Comment by brianhlin** [comment]
@matyasselmeci need anything else?

File: src/README.md [hunk 1]
```diff
@@ -19,6 +19,12 @@ XML consumers
   - [Topology](https://github.com/opensciencegrid/gracc-request/blob/83f3fab52b108b872009430773ce8f1a9fcbe659/config/gracc-request.toml#L41)
   - [VOs](https://github.com/opensciencegrid/gracc-request/blob/83f3fab52b108b872009430773ce8f1a9fcbe659/config/gracc-request.toml#L40)
 - OASIS (OSG)
+  - Both the stratum-0 (oasis.opensciencegrid.org) and stratum-1 (oasis-replica.opensciencegrid.org) query the XML data
+    for `OASISRepoURLs`
+  - `oasis-login` queries for the list of OASIS managers as well as the `UseOASIS` field using the following:
+
+        /vosummary/xml?summary_attrs_showoasis=on&oasis=on&oasis_value=1.
```
Comment (matyasselmeci) [inline src/README.md]: Maybe this should be a full URL?

**Pull request merged**